### DISCUSSION TITLE
Maps with different scenarios

### DISCUSSION THREAD

**Discussion by romanmatus** [carla-simulator/carla#7205] [Q&A]
Hello,

I would like to ask if somebody already created maps with different scenarios.

For example:
- snow on roads
- mud on roads
- country roads / lines
- hill roads
- potholes
..etc.

If yes, could you please share them?
Is this even possible to do ?

Thank you :)

**Comment by haabe** [comment]
You could take a look at the [scenario ChangingWeather from Scenario Runner](https://github.com/carla-simulator/scenario_runner/blob/master/srunner/examples/ChangingWeather.xosc).

Example:
```
<EnvironmentAction>
  <Environment name="Environment1">
    <TimeOfDay animation="true" dateTime="2020-01-01T12:00:00"/>
    <Weather cloudState="free">
      <Sun intensity="0.85" azimuth="0" elevation="1.31"/>
      <Fog visualRange="100000.0"/>
      <Precipitation precipitationType="dry" intensity="0.0"/>
    </Weather>
    <RoadCondition frictionScaleFactor="1.0"/>
  </Environment>
</EnvironmentAction>
```

**Reply by NnaYelsel** [reply]
I don't think that snow is implemented yet.
